devops/scripts/benchmarks/benches/compute.py

```diff
@@ -355,6 +355,39 @@ def createTorchMultiQueueBench(variant_name: str, **kwargs):
                 ),
             ]
 
+        # Add TorchSlmSize benchmarks
+        for runtime in filter(lambda x: x != RUNTIMES.UR, RUNTIMES):
+
+            def createTorchSlmSizeBench(variant_name: str, **kwargs):
+                return TorchSlmSize(
+                    self,
+                    runtime,
+                    variant_name,
+                    PROFILERS.TIMER,
+                    **kwargs,
+                )
+
+            benches += [
+                createTorchSlmSizeBench(
+                    "small",
+                    batchSize=512,
+                    slmNum=1,
+                    warmupIterations=10,
+                ),
+                createTorchSlmSizeBench(
+                    "medium",
+                    batchSize=512,
+                    slmNum=1024,
+                    warmupIterations=5,
+                ),
+                createTorchSlmSizeBench(
+                    "max",
+                    batchSize=512,
+                    slmNum=-1,
+                    warmupIterations=2,
```

Comment:
I'm not sure if we are interested in having scenarios with varying warmup iterations. I would go for one value that makes results variance minimal.